src/cases/containers/case-viewer-container/case-viewer-container.component.ts

```diff
@@ -156,7 +170,10 @@ export class CaseViewerContainerComponent implements OnInit {
       // @ts-ignore
       map(([feature, userRoles, supportedServices, excludedRoles]: [WAFeatureConfig, string[]]) =>
         this.enablePrependedTabs(feature, userRoles, supportedServices, excludedRoles) ? this.prependedTabs : []),
-      catchError(() => this.prependedTabs$ = of([]))
+      catchError((error) => {
+        this.loggerService.log('case-viewer-container - Error in prependedCaseViewTabs', error);
+        return this.prependedTabs$ = of([]);
```

Comment:
The return situation in this function is a bit weird - it's supposed to return the prepended tabs, but it doesn't  explicitly do so (unless there is an error), but it does update this.prependedTab$
I think you should sort this out - make it always return an Observable, but not update the member attribute, and make the calling function do that. Either that, or stop it returning the tabs and make sure that it is solely responsible for setting this.prependedTab$. At the moment you've got both

Reply:
I made the change to set this.prependedTab$ in the method.  To be consistent I made a similar change to setAppendedCaseViewTabs().